[2025-03-12T21:34:31.065+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T21:34:31.089+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.transform_raw_to_staging manual__2025-03-12T21:29:08.864088+00:00 [queued]>
[2025-03-12T21:34:31.102+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.transform_raw_to_staging manual__2025-03-12T21:29:08.864088+00:00 [queued]>
[2025-03-12T21:34:31.103+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T21:34:31.119+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform_raw_to_staging> on 2025-03-12 21:29:08.864088+00:00
[2025-03-12T21:34:31.126+0000] {standard_task_runner.py:72} INFO - Started process 320 to run task
[2025-03-12T21:34:31.130+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'transform_raw_to_staging', 'manual__2025-03-12T21:29:08.864088+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmp3fj319t1']
[2025-03-12T21:34:31.132+0000] {standard_task_runner.py:105} INFO - Job 59: Subtask transform_raw_to_staging
[2025-03-12T21:34:31.153+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T21:34:31.194+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.transform_raw_to_staging manual__2025-03-12T21:29:08.864088+00:00 [running]> on host ***-scheduler
[2025-03-12T21:34:31.322+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='transform_raw_to_staging' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T21:29:08.864088+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T21:29:08.864088+00:00'
[2025-03-12T21:34:31.326+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T21:34:31.357+0000] {manage_buckets.py:46} INFO - [Data manager] Prefix(s) : ['test', 'train', 'validation']
[2025-03-12T21:34:31.358+0000] {manage_buckets.py:48} INFO - [Data manager] Processessing data from : test
[2025-03-12T21:34:31.358+0000] {aws_s3_repository.py:35} INFO - [AWS S3] Getting list of raw's files...
[2025-03-12T21:34:31.372+0000] {manage_buckets.py:48} INFO - [Data manager] Processessing data from : train
[2025-03-12T21:34:31.373+0000] {aws_s3_repository.py:35} INFO - [AWS S3] Getting list of raw's files...
[2025-03-12T21:34:31.385+0000] {manage_buckets.py:48} INFO - [Data manager] Processessing data from : validation
[2025-03-12T21:34:31.406+0000] {aws_s3_repository.py:35} INFO - [AWS S3] Getting list of raw's files...
[2025-03-12T21:34:31.413+0000] {manage_buckets.py:59} ERROR - [Data manager] Data processing failed.
[2025-03-12T21:34:31.414+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-12T21:34:31.434+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T21:34:31.436+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=transform_raw_to_staging, run_id=manual__2025-03-12T21:29:08.864088+00:00, execution_date=20250312T212908, start_date=20250312T213431, end_date=20250312T213431
[2025-03-12T21:34:31.510+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T21:34:31.550+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T21:34:31.555+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
